gpkit/solution_array.py

```diff
@@ -0,0 +1,99 @@
+import numpy as np
+
+from .posyarray import PosyArray
+from .small_classes import Strings
+from .small_classes import DictOfLists
+from .small_scripts import results_table
+from .small_scripts import unitstr
+from .small_scripts import mag
+
+
+class SolutionArray(DictOfLists):
+    "DictofLists extended with posynomial substitution."
```

Comment:
I think this docstring should be expanded to include the keys we typically find in a SolutionArray (since many other methods and classes rely on that API).